app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -818,3 +817,5 @@ def cachekey_with_cluster(key, cluster_id = nil)
     end
   end
 end
+
+require 'fog_extensions/vsphere/mini_servers'
```

Comment:
Question - why this change? Is there a difference between `require` at the beginning of file and at the end of the file?

Reply:
This is weird... At the time I've been writing this, I've faced circular dependency in loading `Vmware` constant, but just to remember why exactly I did that, I've put this line back, run the `zeitwerk:check` again and all seems good. So, indeed, there is no need to put this line to the end :/ Reverted.